cli/Makefile

```diff
@@ -42,7 +42,7 @@ turborepo-ffi-install: turborepo-ffi turborepo-ffi-copy-bindings
 
 .PHONY: turborepo-ffi
 turborepo-ffi:
-	cd ../crates/turborepo-ffi && cargo build --release --target-dir ./target
+	cd ../crates/turborepo-ffi && cargo build --target-dir ./target
```

Comment:
how/where is release mode configured?

Reply:
There are other tasks that are used during cross-compile that do use release mode. See https://github.com/vercel/turbo/blob/2d979d2f42e30322695a333373b35b5908196df5/cli/Makefile#L79-L97